docs/Team.md

```diff
@@ -80,7 +80,8 @@ Area Lead for: UI [2016 Aug - 2017 May]
 
 #### [Leow Yijin](http://github.com/yijinl)
```

Comment:
On an unrelated note (not related to this PR), this link is broken. Did the person changed his alias/deleted his account?